schema.json

```diff
@@ -5,6 +5,10 @@
   "$defs": {
     "Config": {
       "properties": {
+        "$schema": {
```

Comment:
unfortunately this will not work, as this file is auto-generated from the configuration.

will take a look on how to make this permanent.